wolfssl/wolfcrypt/asn.h

```diff
@@ -1383,7 +1383,7 @@ struct SignatureCtx {
 #endif
 #if !defined(NO_RSA) || !defined(NO_DSA)
     #ifdef WOLFSSL_NO_MALLOC
-    byte  sigCpy[MAX_SIG_SZ];
```

Comment:
This leaves `MAX_SIG_SZ` dead (not used). Should we leave it or remove it?

Reply:
PR Cap @JacobBarthelmeh thoughts?

Reply:
My thoughts are it should be removed if not used, to avoid confusion in the future.